src/components/Common/Header/index.tsx

```diff
@@ -31,14 +33,27 @@ const Header: React.FC = () => {
             <S.NavContent css={select('/faq')}>자주 묻는 질문</S.NavContent>
           </Link>
         </S.NavBar>
-        <S.MemberBox>
-          <Link href="/auth/signin" passHref>
-            <S.MemberContent css={select('/signin')}>로그인</S.MemberContent>
-          </Link>
-          <Link href="/auth/signup" passHref>
-            <S.MemberContent css={select('/signup')}>회원가입</S.MemberContent>
-          </Link>
-        </S.MemberBox>
+        {!logged ? (
+          <S.MemberBox>
+            <Link href="/auth/signin" passHref>
+              <S.MemberContent css={select('/auth/signin')}>로그인</S.MemberContent>
+            </Link>
+            <Link href="/auth/signup" passHref>
+              <S.MemberContent css={select('/auth/signup')}>
+                회원가입
+              </S.MemberContent>
+            </Link>
+          </S.MemberBox>
+        ) : (
+          <S.MemberBox>
+            <Link href="/mypage" passHref>
+              <S.MemberContent css={select('/mypage')}>프로필</S.MemberContent>
```

Comment:
```suggestion
              <S.MemberContent css={select('/mypage')}>내 정보</S.MemberContent>
```
디자인에선 '내 정보'라고 나와있네요

Reply:
놓친 부분인데 캐치해주셔서 감사합니다!